scripts/install/install-pocs.sh

```diff
@@ -3,7 +3,7 @@ set -e
 
 usage() {
   echo -n "##################################################
-# Install POCS and friends.
+# Install POCS l friends.
```

Comment:
I must have accidentally changed 'and' to 'l'. Disregard this.

Reply:
There are a whole bunch of these and they'll need to be changed back in the PR. I can't selectively pick parts.